std_msgs/Int32 id
---
std_msgs/String name
std_msgs/Bool found_name